### PR TITLE
[GEOT-5595] Cannot read/write GeoTIFF with Azimuthal_Equidistant projection (master)

### PR DESCRIPTION
**Pull request by bencaradocdavies**
https://osgeo-org.atlassian.net/browse/GEOT-5595

Note that this PR depends on and incorporates the commit in https://github.com/geotools/geotools/pull/1421

### CODE REVIEW

File: modules/library/referencing/src/test/java/org/geotools/referencing/operation/projection/AzimuthalEquidistantTest.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/*
```
Comment (simboss) [inline modules/library/referencing/src/test/java/org/geotools/referencing/operation/projection/AzimuthalEquidistantTest.java]: Are you sure this change should be here?

Isn't this part of the other PR?

File: modules/library/referencing/src/main/java/org/geotools/referencing/operation/projection/AzimuthalEquidistant.java [hunk 2]
```diff
@@ -152,6 +152,20 @@ public ParameterDescriptorGroup getParameterDescriptors() {
             return Provider.PARAMETERS;
```
Comment (simboss) [inline modules/library/referencing/src/main/java/org/geotools/referencing/operation/projection/AzimuthalEquidistant.java]: Are you sure this change should be here?

Isn't this part of the other PR?

**Comment by bencaradocdavies** [comment]
@simboss, please see the description; this one relies on https://github.com/geotools/geotools/pull/1421 and the merge will fix all.  :-)

**Comment by bencaradocdavies** [comment]
@simboss that worked as expected: the GEOT-5594 commit does not appear in the merge as viewed in gitk. Apologies for the confusion; just keeping Travis CI happy.

**Pull request merged**